mongo/integration/unified/context.go

```diff
@@ -29,7 +34,13 @@ const (
 
 // newTestContext creates a new Context derived from ctx with values initialized to store the state required for test
 // execution.
-func newTestContext(ctx context.Context, entityMap *EntityMap, expectedLogMessageCount int) context.Context {
+func newTestContext(
+	ctx context.Context,
+	entityMap *EntityMap,
+	expectedLogMessageCount int,
+	hasOperationalFailPoint bool,
+) context.Context {
+	ctx = context.WithValue(ctx, operationalFailPointKey, hasOperationalFailPoint)
```

Comment:
Before this ticket we supported reducing heartbeatFrequencyMS at the file-level client, but not the operational-level client. Since the operation does not maintain access to the "TestCase" object, it feels natural to extend the test context to include whether or not the set of operations contains a fail point operation.